SQL/SQL.sublime-syntax

```diff
@@ -259,12 +306,28 @@ contexts:
       pop: true
     - match: (?=\S)
       pop: true
-  like_string:
-    - match: \'
-      scope: punctuation.definition.string.begin.sql
-      set: inside_like_single_quoted_string
+  like_escape_slash:
```

Comment:
The name of those contexts conflicts with `like_escape_fail` and `like_escape_pop` somehow as those handle behavior on `ESCAPE` keywords while this one is about the ecape character.

Maybe this difference in meaning should be reflected by the contexts' names?